### PR TITLE
Piwik custom tracking

### PR DESCRIPTION
**Pull request by Langleu**

### CODE REVIEW

File: static/scripts/piwik/piwik.js [hunk 1]
```diff
@@ -1,6 +1,15 @@
 var _paq = _paq || [];
-_paq.push(['trackPageView']);
-_paq.push(['enableLinkTracking']);
+$.get('/account/user', (res) => {
+    if (typeof (res || {}).firstName === 'string') {
```
Comment (jprberlin) [inline static/scripts/piwik/piwik.js]: is this how to detect if the user is logged in? If so add an comment

File: static/scripts/piwik/piwik.js [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+        _paq.push(['setCustomDimension', customDimensionId = 2, customDimensionValue = res.gender]);
+        _paq.push(['setCustomDimension', customDimensionId = 4, customDimensionValue = res.roles[0].name]);
+    }
+
+    _paq.push(['trackPageView']);
+    _paq.push(['enableLinkTracking']);
+});
+
 (function () {
     var u = "//open.hpi.de/piwik/";
```
Comment (jprberlin) [inline static/scripts/piwik/piwik.js]: should this go to some global config?

Comment (Langleu) [inline static/scripts/piwik/piwik.js]: no

File: static/scripts/piwik/piwik.js (hunk 1 above)
Comment (Langleu) [inline static/scripts/piwik/piwik.js]: will change it a bit

**Pull request merged**